_posts/2020-09-07-sparklyr-flint/sparklyr_flint.Rmd

```diff
@@ -0,0 +1,227 @@
+---
+title: "Introducing sparklyr.flint: A time-series extension for sparklyr"
+description: |
+  We are pleased to announce sparklyr.flint, a sparklyr extension for analyizing time series at scale, is now available on CRAN.
+author:
+  - name: Yitao Li
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com
+slug: sparklyr.flint-0.1.1
+date: 09-07-2020
+categories:
+  - R
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+In this blog post, we will showcase [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a [`sparklyr`](https://sparklyr.ai) extension providing a simple and intuitive R interface to the [`Flint`](https://github.com/twosigma/flint) time series library.
+
```

Comment:
We have this tradition of encouraging people to install from CRAN with R code and also, worth mentioning is in CRAN in the first paragraph :)

``````suggestion
In this blog post, we will showcase [`sparklyr.flint`](https://cran.r-project.org/web/packages/sparklyr.flint/index.html), a brand new [`sparklyr`](https://sparklyr.ai) extension providing a simple and intuitive R interface to the [`Flint`](https://github.com/twosigma/flint) time series library. `sparklyr.flint` is available on [CRAN](https://cran.r-project.org/web/packages/sparklyr.flint/index.html) today and can be installed as follows:

```r
install.packages("sparklyr.flint")
```

``````